templates/site/usage.html

```diff
@@ -0,0 +1,56 @@
+{% extends "layouts/main.html" %}
+{% set page_title = "Marble Users" %}
+{% block content %}
+    {% include "partials/banner.html" %}
+
+       {% with %}
+            {% set img_src = "images/personas/persona-mark.png" %}
+            {% set img_class = "img-fluid" %}
+            {% set img_location = "left" %}
+            {% set second_img_src = "images/personas/grad-student-secondary-image-transparentbg-640.png" %}
+            {% set second_img_class = "img-fluid" %}
+            {% set second_img_location = "right" %}
+            {% set title = "Grad Students" %}
+            {% set intro_content = "" %}
+            {% set content %}
+                <p>Marble provides an all-in-one data catalog and development environment hosted by U of T or another
+                    member of the Marble network.  Data stored on the Marble network can be accessed by all users, no
+                    matter where it is hosted.  Using web processing services tailored for climate analysis, Marble
+                    is great for those getting started in climate research. </p>
+
+                <p>Marble uses JupyterLab as its development environment.  JupyterLab lets you create self-contained
+                    notebooks that can access datasets stored on Marble's STAC catalog, and conduct data analysis using tools such as
+                    data science oriented Python packages.</p>
+
+                <p>All data storage and data processing are done on the Marble platform.  No need to
+                    ensure your laptop has enough storage or computing power.</p>
+            {% endset %}
+            {% include "partials/info-row.html" %}
+        {% endwith %}
+
+       {% with %}
+            {% set img_src = "images/personas/persona-michelle.png" %}
+            {% set img_class = "img-fluid" %}
+            {% set img_location = "right" %}
+            {% set second_img_src = "images/personas/researcher-secondary-image-transparentbg-640.png" %}
+            {% set second_img_class = "img-fluid" %}
+            {% set second_img_location = "left" %}
+            {% set title = "University Researcher" %}
```

Comment:
Reading the blurb below, it am not sure how this differs from a "grad student". All graduate students are "university researchers" and a majority of "university researchers" are "grad students". I feel we might be able to consolidate these two.

Reply:
I can see how they can be very similar since this is supposed to highlight the attractive qualities of Marble for both these roles.

The grad student persona is a new grad student who likely hasn't worked with any data science tools.  The university researcher persona is someone who is already doing research.  This is trying to get the university researcher onto a new tool.

Reply:
The distinction makes sense to me, but perhaps call the second one "experienced researcher"?

Reply:
That would work and it would remove the need to make another researcher type.  One of the new personas I made was Government Researcher, and it would be 90% the same as University Researcher.  I'll see if I can work something in to this entry to fill the needs of Government Researcher as well.